### PR TITLE
Update Stripe redirect & add holding message

### LINKED ISSUE #1955: Redirect back from Stripe takes you to 404 page
### Current Behavior

On staging, after creating a new team and setting up billing on stripe, you are redirected back to a 404 page.

`https://forge.flowforge.dev/team/demo-team-2/overview?billing_session=*****`

This is because the team 'home' is no longer `/overview`

### Expected Behavior

To be taken to a page that doesn't 404

### Steps To Reproduce

Create a new team, head to stripe to setup billing, see where you return.

### Environment

- FlowForge version: 1.6 pre release on staging at time of raising issue
- Node.js version:
- npm version:
- Platform/OS:
- Browser:

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- Directs users to `/applications` instead of `/overview` which no longer exists
- Adds a holding message which is _very_ unlikely to be seen, as they join in trial, and setup an Application (for free) as part of the onboarding, but it's nice to be nice just in case this state does arise.

Screenshot 2023-04-11 at 18 45 46

## Related Issue(s)

Close #1955 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by joepavitt** [comment]
I've seen this test failing randomly on a few occasions now, it should be resolved and fixed properly. Will open a separate PR for it.

```
⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  test/unit/frontend/utils/elapsedTime.spec.js > elapsedTime > compares relative to now if only a to date is passed
AssertionError: expected '30 minutes, 14 seconds' to be '30 minutes, 15 seconds' // Object.is equality
 ❯ test/unit/frontend/utils/elapsedTime.spec.js:58:35
     56|         soon.setSeconds(soon.getSeconds() + 15)
     57| 
     58|         expect(elapsedTime(soon)).toBe('30 minutes, 15 seconds')
       |                                   ^
     59|     })
 Test Files  1 failed | 7 passed (8)
     60| 
      Tests  1 failed | 56 passed (57)

   Start at  17:49:42
  - Expected  - 1
   Duration  9.78s (transform 673ms, setup 1ms, collect 1.29s, tests 333ms, environment 4.48s, prepare 946ms)
  + Received  + 1


  - '30 minutes, 15 seconds'
  + '30 minutes, 14 seconds'
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯
```

File: frontend/src/pages/team/Applications.vue [hunk 1]
```diff
@@ -187,6 +195,9 @@ export default {
                     id: instance.id
                 }
             })
+        },
+        async checkBillingSession () {
```
Comment (Pezmc) [inline frontend/src/pages/team/Applications.vue:199]:
```suggestion
       checkBillingSession () {
```

File: frontend/src/pages/team/Applications.vue [hunk 2]
```diff
@@ -187,6 +195,9 @@ export default {
...
+            this.justSetupBilling = 'billing_session' in this.$route.query
```
Comment (Pezmc) [inline frontend/src/pages/team/Applications.vue:200]: This should be a computed property

**Review by Pezmc — APPROVED** [review]

**Comment by knolleary** [comment]
> Adds a holding message which is very unlikely to be seen, as they join in trial, and setup an Application (for free) as part of the onboarding, but it's nice to be nice just in case this state does arise.

Setting up an application isn't part of onboarding. We setup the team for them without billing. If they choose to configure billing before they create their trial application then they will see this message.

They will also see this message if they choose to create a second team.

**Pull request merged**